tasks.py

```diff
@@ -1,70 +1,84 @@
+import os
+from pathlib import Path
+
+import requests
 from robocorp import browser
 from robocorp.tasks import task
 from RPA.Excel.Files import Files as Excel
 
-from pathlib import Path
-import os
-import requests
-
 FILE_NAME = "challenge.xlsx"
-OUTPUT_DIR = Path(os.environ.get('ROBOT_ARTIFACTS'))
 EXCEL_URL = f"https://rpachallenge.com/assets/downloadFiles/{FILE_NAME}"
+OUTPUT_DIR = Path(os.getenv("ROBOT_ARTIFACTS", "output"))
 
 
 @task
 def solve_challenge():
-    """
-    Solve the RPA challenge
-    
-    Downloads the source data excel and uses Playwright to solve rpachallenge.com from challenge
+    """Main task which solves the RPA challenge!
+
+    Downloads the source data Excel file and uses Playwright to fill the entries inside
+    rpachallenge.com.
     """
     browser.configure(
-        browser_engine="chromium",
-        screenshot="only-on-failure",
-        headless=True,
+        browser_engine="chromium", screenshot="only-on-failure", headless=True
     )
     try:
+        # Reads a table from an Excel file hosted online.
         excel_file = download_file(EXCEL_URL, OUTPUT_DIR, FILE_NAME)
         excel = Excel()
         excel.open_workbook(excel_file)
         rows = excel.read_worksheet_as_table("Sheet1", header=True)
 
+        # Surf the automation challenge website and fill in information from the table
+        #  extracted above.
         page = browser.goto("https://rpachallenge.com/")
         page.click("button:text('Start')")
         for row in rows:
-            fill_and_submit_form(row)
+            fill_and_submit_form(row, page=page)
         element = page.locator("css=div.congratulations")
         browser.screenshot(element)
-
     finally:
-        # Place for teardown and cleanups
-        # Playwright handles browser closing
-        print('Done')
+        # A place for teardown and cleanups. (Playwright handles browser closing)
+        print("Automation finished successfully!")
+
+
+def download_file(url: str, target_dir: Path, target_filename: str) -> Path:
+    """Downloads a file from the given URL into a custom folder & name.
```

Comment:
Saving one linefeed here "hurts my soul", if the documentation is multiline having the content starting from same indentation level makes things a lot more readable. 
One of the things Google Style does not really state exactly but the way I read it the same line comment is only allowed when the doc is single line and under 80 chars.

Reply:
Oh, so it's a readability issue when we read the docstring in VSCode I believe? (because almost in every Python library (beside ours) I saw that the docstring **doesn't** being with an EOL no matter the case)

Shall we diverge from [pep8](https://peps.python.org/pep-0008/#documentation-strings) (and what the extension automatically generated for us without me changing the formatting)?

Reply:
Ok, got it from @fabioz: [pep257](https://peps.python.org/pep-0257/#multi-line-docstrings)